susemanager-utils/susemanager-sls/modules/engines/mgr_events.py

```diff
@@ -93,7 +94,7 @@ def __init__(self, event_bus, config):
         self.config.setdefault('postgres_db', {})
         self.config['postgres_db'].setdefault('host', 'localhost')
         self.config['postgres_db'].setdefault('notify_channel', 'suseSaltEvent')
-        self.counter = 0
+        self.counters = [0 for i in range(-1, config['events']['thread_pool_size'])]
```

Comment:
Shouldn't this be from 0 to thread_pool_size + 1 as well?

Reply:
sure... just a leftover

Reply:
:heavy_check_mark: